src/GraphQL.ApiTests/ApiApprovalTests.PublicApi.GraphQL.approved.txt

```diff
@@ -2678,12 +2678,6 @@ namespace GraphQL.Validation.Rules
         public string InlineFragmentOnNonCompositeErrorMessage(string type) { }
         public System.Threading.Tasks.Task<GraphQL.Validation.INodeVisitor> ValidateAsync(GraphQL.Validation.ValidationContext context) { }
     }
-    public static class ISelectionExtensions
```

Comment:
These methods essentially were originally supposed to be internal since they are used only by one of the validation rules.